pkg/receive/handler.go

```diff
@@ -456,6 +487,21 @@ func (h *Handler) receiveHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Impose limits only if Receive is in Router or RouterIngestor mode.
+	if h.receiverMode == RouterOnly || h.receiverMode == RouterIngestor {
```

Comment:
Curious, why can't we have this feature in the dual mode as well?

Reply:
By dual mode, you mean why not also in IngestorOnlyr?

Reply:
Ah yes, dual mode seems to be `RouterIngestor`. I guess we don't need limiting for `IngestorOnly` because the router will tackle that?

Reply:
Yes exactly. 🙂

Reply:
Got it, thanks 👍